core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/SurveyFormatter.java

```diff
@@ -77,7 +76,13 @@ private void generateAnswerItemFormatters(
         )).values().stream().sorted(Comparator.comparingInt(a -> a.get(0).getExportOrder())).toList();
 
         for (List<SurveyQuestionDefinition> questionVersions : questionDefsByStableId) {
-            SurveyQuestionDefinition mostRecent = questionVersions.get(0);
+
+            Optional<SurveyQuestionDefinition> mostRecentOpt = getLatest(questionVersions);
```

Comment:
Instead of doing this, I think it's cleaner to update the code above to sort the lists as they're made.  

I think you can pass a second argument to groupingBy of a collector that sorts the sublists. See https://stackoverflow.com/questions/35872236/sorting-lists-after-groupingby